server/routerlicious/packages/lambdas/src/alfred/interfaces.ts

```diff
@@ -0,0 +1,70 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IEvent } from "@fluidframework/common-definitions";
+import { IClient, IConnected } from "@fluidframework/protocol-definitions";
+
+/**
+ * Connection details of a client
+ */
+export interface IConnectedClient {
+	/**
+	 *
+	 */
+	connection: IConnected;
+
+	/**
+	 *
+	 */
+	details: IClient;
```

Comment:
@znewton @hedasilv @pradeepvairamani same here with docs for details. 
IClient docs are:
```
/**
 * Represents a client connected to a Fluid service, including associated user details, permissions, and connection mode.
 */
 ```

Reply:
It could be worth mentioning that this is information is populated with a combination of the connect message's  `IConnect.client` property and the claims parsed from the `IConnect.token` property.

```
/**
 * Connected client details including associated user details, permissions, and connection mode.
 * Most details come from the {@link IConnect.client} message property, but {@link IClient.user} and {@link IClient.scopes} properties are overwritten by the parsed claims from the validated {@link IConnect.token}.
 */
```